CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [2.18.0](https://github.com/appium/appium-uiautomator2-driver/compare/v2.17.0...v2.18.0) (2023-04-20)
```

Comment:
this change is not needed since a release action will do this automatically by checking the commit message

Reply:
oh, sorry. this is current master's one